vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**nitpick**: Consider using semicolons consistently to maintain code style uniformity.